CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+
+## Bigger changes
+
+If you want to make a bigger change, it's a good idea to first file an issue and make sure someone from the team agrees that it’s needed. 
+If you’ve found a bug, please file an issue that illustrates the bug with a minimal 
+[reprex](https://www.tidyverse.org/help/#reprex) (this will also help you write a unit test, if needed).
+See the Tidyverse guide on [how to create a great issue](https://code-review.tidyverse.org/issues/) for more advice.
+
+### Pull request process
+
+*   Pull the latest version of `main` and install all dependencies with `renv::restore()`.
+*   Create a Git branch for your pull request (PR). You could use `usethis::pr_init("brief-description-of-change")`.
+*   Make your changes, commit to git, and then create a PR by running `usethis::pr_push()`, and following the prompts in your browser.
+    Ensure that `CHANGES.md` is updated as described below.
+    The title of your PR should briefly describe the change.
+    The body of your PR should contain `Fixes #issue-number`.
+
+### Update `CHANGES.md`
+
+*   All PR changes need to be reflected in `CHANGES.md`. Add a bullet to the top
+of `NEWS.md` (i.e. just below the first header). Follow the style described
+in <https://style.tidyverse.org/news.html>.
+*   Since there is only a single branch `main`, every PR will need to increment
+the version number as described in the next section.
+*  `CHANGES.md` is automatically parsed to create a public-facing `NEWS.md` in
+the methods documentation. This is done by ensuring `CHANGES.md` is formatted
+like so:
+
+```
+## version number
+
+### Section
+*   Change 1
+*   Change 2
+```
+
+`Section` should follow the below guide to ensure proper parsing to `NEWS.md`.
+
+Public facing:
+
+*   `indicator_id`: (e.g. `ipc_food_insecurity` or `jrc_agricultural_hotspots`) to
+indicate any change to code or emails specific to that indicator code that impacts
+methods for scanning the data, analyzing and visualizing it, or generating email.
+Does not include code improvements or other changes that are not necessary to
+report to users of Signals.
+*   `Email`: changes to the email, including plots, maps, or text, that are not
+specific to a single indicator.
+*   `Data`: changes to the output dataset `hdx_signals.csv`.
+*   `Metadata`: changes to the locations metadata data `hdx_signals_locations_metadata.csv`.
+
+Non-public facing:
+
+*   `System`: changes in the `src` code that are not necessary to reflect publicly,
+which can include improvements or other changes in the indicators folder.
+*   `Assets`: changes to the `src-static` code to generate static assets.
+*   `Testing`: changes to the `test` folder.
+*   `Monitoring`: changes to the `.github` actions files that change how we monitor.
+
+Feel free to propose additional categories, but those need to be reflected here for
+consistency.
+
+### Versioning
```

Comment:
And here, something like: 

"Update the version in `.signals-version`, following semantic versioning guidelines."